fdbbackup/FileDecoder.actor.cpp

```diff
@@ -421,13 +412,27 @@ class DecodeProgress {
 		if (!mutationBlocksByVersion.empty()) {
 			TraceEvent(SevWarn, "UnfishedBlocks").detail("NumberOfVersions", mutationBlocksByVersion.size());
 		}
-		done = true;
-		return vms;
+		return Optional<VersionedMutations>();
 	}
 
 	ACTOR static Future<Void> openFileImpl(DecodeProgress* self, Reference<IBackupContainer> container) {
 		Reference<IAsyncFile> fd = wait(container->readFile(self->file.fileName));
 		self->fd = fd;
+		if (self->save) {
```

Comment:
I guess this is not relevant to knobs?

Reply:
Correct. This is adding an option to save backup files. Maybe reviewing by commits is easier.